### PR TITLE
Zib2020 CareAgreement

### PR DESCRIPTION
**Pull request by niekvangalen**

### CODE REVIEW

File: ada_2_fhir-r4/zibs2020/payload/0.7-dev/nl-core-CareAgreement.xsl [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+                        </xsl:if>
+                        <xsl:if test="patient">
+                            <xsl:call-template name="makeReference">
+                                <xsl:with-param name="in" select="$careCoordinator"/>
+                                <xsl:with-param name="profile" select="'nl-core-Patient'"/>
+                            </xsl:call-template>
+                        </xsl:if>
+                    </author>
+                </xsl:for-each>
+                <xsl:for-each select="$agreementParties">
```
Comment (LuudSlagter) [inline ada_2_fhir-r4/zibs2020/payload/0.7-dev/nl-core-CareAgreement.xsl:98]: Here I'd expect the if statements above. In line 101 `$agreementParties` should be replaced by `.`.

File: ada_2_fhir-r4/zibs2020/payload/0.7-dev/nl-core-CareAgreement.xsl [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+                            <status>
+                                <xsl:attribute name="value" select="'in-progress'"/>
+                            </status>
+                        <xsl:call-template name="makeReference">
+                            <xsl:with-param name="in" select="$performer"/>
+                            <xsl:with-param name="profile" select="'nl-core-HealthProfessional-PractitionerRole'"/>
+                            <xsl:with-param name="wrapIn" select="'performer'"/>
+                        </xsl:call-template>
+                        <xsl:for-each select="toelichting">
+                            <description value="{normalize-space(@value)}"/>
```
Comment (LuudSlagter) [inline ada_2_fhir-r4/zibs2020/payload/0.7-dev/nl-core-CareAgreement.xsl:128]: Maybe use `string-to-string` here as well (I think we need to make this consistent over all stylesheets, actually).

File: ada_2_fhir-r4/zibs2020/payload/0.7-dev/nl-core-CareAgreement.xsl [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+                </xsl:call-template>
+                <activity>
+                    <detail>
+                        <xsl:for-each select="activiteit">
+                            <code>
+                                <text value="{@value}"/>
+                            </code>
+                            </xsl:for-each>
+                            <!-- Follows implicit zib mapping to StartDate and EndDate as described in the profile, 'in-progress' == 'active'. -->
+                            <status>
```
Comment (LuudSlagter) [inline ada_2_fhir-r4/zibs2020/payload/0.7-dev/nl-core-CareAgreement.xsl:119]: You could just use `<status value="in-progress"/>`, see remark on `status` above.

File: ada_2_fhir-r4/zibs2020/fhir_instance/nl-core-CareAgreement-01-Condition-01.xml [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+         <system value="http://terminology.hl7.org/CodeSystem/condition-ver-status"/>
+         <code value="confirmed"/>
+         <display value="Confirmed"/>
+      </coding>
+      <coding>
+         <system value="http://snomed.info/sct"/>
+         <code value="410605003"/>
+         <display value="Aanwezigheid bevestigd"/>
+      </coding>
+   </verificationStatus>
```
Comment (Ashwin-nictiz) [inline ada_2_fhir-r4/zibs2020/fhir_instance/nl-core-CareAgreement-01-Condition-01.xml:25]: The QA tool gives the following error:

```
-  error at Condition.verificationStatus (14, 24):
     Condition.verificationStatus.coding:verificationStatusCodelist: minimum required = 1, but only found 0 (from http://nictiz.nl/fhir/StructureDefinition/nl-core-Problem)
```

The code is included in the example; perhaps a QA error?

File: ada_2_fhir-r4/zibs2020/fhir_instance/nl-core-CareAgreement-01.xml [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+         </code>
+         <status value="in-progress"/>
+         <performer>
+            <reference value="Patient/nl-core-CareAgreement-01-Patient-01"/>
+            <type value="Patient"/>
+            <display value="Patient, A.S. van Dongen"/>
+         </performer>
+         <description value="POH-GGZ van de huisartspraktijk leest en reageert wekelijks (indien nodig)."/>
+      </detail>
+   </activity>
```
Comment (Ashwin-nictiz) [inline ada_2_fhir-r4/zibs2020/fhir_instance/nl-core-CareAgreement-01.xml:47]: The QA tools gives the following error:

  -  error at CarePlan.category[0] (9, 14):
     Slicing cannot be evaluated: Could not match discriminator ([coding]) for slice CarePlan.category:careAgreement in profile http://nictiz.nl/fhir/StructureDefinition/nl-core-CareAgreement - the discriminator [coding] does not have fixed value, binding or existence assertions
  -  error at CarePlan.contributor[0] (24, 17):
     Slicing cannot be evaluated: Problem evaluating slicing expression for element in profile http://nictiz.nl/fhir/StructureDefinition/nl-core-CareAgreement path CarePlan.contributor[0] (fhirPath = true and resolve().conformsTo('http://nictiz.nl/fhir/StructureDefinition/nl-core-ContactPerson')): The URL 'PractitionerRole/nl-core-CareAgreement-01-PractitionerRole-01' is not known to the FHIR validator, and has not been provided as part of the setup / parameters
  -  error at CarePlan.contributor[0] (24, 17):
     Slicing cannot be evaluated: Problem evaluating slicing expression for element in profile http://nictiz.nl/fhir/StructureDefinition/nl-core-CareAgreement path CarePlan.contributor[0] (fhirPath = true and resolve().conformsTo('http://nictiz.nl/fhir/StructureDefinition/nl-core-HealthProfessional-PractitionerRole')): The URL 'PractitionerRole/nl-core-CareAgreement-01-PractitionerRole-01' is not known to the FHIR validator, and has not been provided as part of the setup / parameters
  -  error at CarePlan.contributor[0] (24, 17):
     Slicing cannot be evaluated: Problem evaluating slicing expression for element in profile http://nictiz.nl/fhir/StructureDefinition/nl-core-CareAgreement path CarePlan.contributor[0] (fhirPath = true and resolve().conformsTo('http://nictiz.nl/fhir/StructureDefinition/nl-core-Patient')): The URL 'PractitionerRole/nl-core-CareAgreement-01-PractitionerRole-01' is not known to the FHIR validator, and has not been provided as part of the setup / parameters
  -  error at CarePlan.activity[0].detail.performer[0] (40, 21):
     Slicing cannot be evaluated: Problem evaluating slicing expression for element in profile http://nictiz.nl/fhir/StructureDefinition/nl-core-CareAgreement path CarePlan.activity[0].detail.performer[0] (fhirPath = true and resolve().conformsTo('http://nictiz.nl/fhir/StructureDefinition/nl-core-ContactPerson')): The URL 'Patient/nl-core-CareAgreement-01-Patient-01' is not known to the FHIR validator, and has not been provided as part of the setup / parameters
```
  -  error at CarePlan.activity[0].detail.performer[0] (40, 21):
     Slicing cannot be evaluated: Problem evaluating slicing expression for element in profile http://nictiz.nl/fhir/StructureDefinition/nl-core-CareAgreement path CarePlan.activity[0].detail.performer[0] (fhirPath = true and resolve().conformsTo('http://nictiz.nl/fhir/StructureDefinition/nl-core-HealthProfessional-PractitionerRole')): The URL 'Patient/nl-core-CareAgreement-01-Patient-01' is not known to the FHIR validator, and has not been provided as part of the setup / parameters
  -  error at CarePlan.activity[0].detail.performer[0] (40, 21):
     Slicing cannot be evaluated: Problem evaluating slicing expression for element in profile http://nictiz.nl/fhir/StructureDefinition/nl-core-CareAgreement path CarePlan.activity[0].detail.performer[0] (fhirPath = true and resolve().conformsTo('http://nictiz.nl/fhir/StructureDefinition/nl-core-Patient')): The URL 'Patient/nl-core-CareAgreement-01-Patient-01' is not known to the FHIR validator, and has not been provided as part of the setup / parameters
```

**Pull request closed without merging**